### PR TITLE
Removing remaining jquery imports

### PR DESCRIPTION
**Pull request by jespino**
This leave the jquery dependency in the package.json because it is required by
bootstrap. We need to eventually remove bootstrap to be able to definitively
get rid of jquery.

#### Summary
Removing remaining jquery imports

#### Ticket Link
https://mattermost.atlassian.net/browse/MM-27444
https://mattermost.atlassian.net/browse/MM-27445
https://mattermost.atlassian.net/browse/MM-29031
https://mattermost.atlassian.net/browse/MM-29032

#### Release Note
```release-note
NONE
```

### CODE REVIEW

File: components/audio_video_preview.jsx [hunk 1]
```diff
@@ -39,7 +37,7 @@ export default class AudioVideoPreview extends React.PureComponent {
         this.handleFileInfoChanged(this.props.fileInfo);
 
         if (this.sourceRef.current) {
-            $(ReactDOM.findDOMNode(this.sourceRef.current)).one('error', this.handleLoadError);
+            this.sourceRef.current.addEventListener('error', this.handleLoadeError, {once: true});
```
Comment (hmhealey) [inline components/audio_video_preview.jsx:40]:
```suggestion
            this.sourceRef.current.addEventListener('error', this.handleLoadError, {once: true});
```

File: components/audio_video_preview.jsx [hunk 2]
```diff
@@ -49,12 +47,12 @@ export default class AudioVideoPreview extends React.PureComponent {
         }
 
         if (this.sourceRef.current) {
-            $(ReactDOM.findDOMNode(this.sourceRef.current)).one('error', this.handleLoadError);
+            this.sourceRef.current.addEventListener('error', this.handleLoadeError, {once: true});
```
Comment (hmhealey) [inline components/audio_video_preview.jsx:50]:
```suggestion
            this.sourceRef.current.addEventListener('error', this.handleLoadError, {once: true});
```

File: components/searchable_user_list/searchable_user_list.jsx [hunk 3]
```diff
@@ -84,14 +82,14 @@ class SearchableUserList extends React.PureComponent {
         this.nextTimeoutId = setTimeout(() => this.setState({nextDisabled: false}), NEXT_BUTTON_TIMEOUT);
 
         this.props.nextPage();
-        $(ReactDOM.findDOMNode(this.refs.channelListScroll)).scrollTop(0);
+        this.refs.userList.scrollToTop();
```
Comment (hmhealey) [inline components/searchable_user_list/searchable_user_list.jsx:85]: Funny that no one ever noticed this didn't work before. I guess jQuery caused that to fail silently

**Review by hmhealey — CHANGES_REQUESTED** [review]
Awesome! I'm glad we're finally getting rid of jquery. It's been on my roadmap for a while to get rid of it. Just a typo from copy-pasting, then this'll be good to go

File: components/audio_video_preview.jsx (hunk 1 above)
Comment (jespino) [reply components/audio_video_preview.jsx:40]: Good catch

File: components/searchable_user_list/searchable_user_list.jsx (hunk 3 above)
Comment (jespino) [reply components/searchable_user_list/searchable_user_list.jsx:85]: Yes, I guess so.

**Review by devinbinnie — APPROVED** [review]
image

**Review by hmhealey — APPROVED** [review]

**Pull request merged**